packages/flutter/lib/src/cupertino/date_picker.dart

```diff
@@ -931,6 +941,49 @@ class _CupertinoDatePickerDateState extends State<CupertinoDatePicker> {
 // layout will be broken.
 
 
+/// Controls a [CupertinoTimerPicker] widget.
+///
+/// Each controller can only be used with a single Cupertino timer picker.
+///
+/// Used with [CupertinoTimerPicker].
+///
+/// See also:
+///
+///  * [CupertinoTimerPicker] which creates a Cupertino themed time picker which
+///    uses this controller.
+///  * [CupertinoDatePicker] which creates a Cupertino themed date picker.
+///
+class CupertinoTimerPickerController extends ChangeNotifier {
```

Comment:
When we first discussed a controller, I thought we were actually going to codify some behavior in the controller, .e.g, `reset()`.  If we're going to implement the controller as just 2 properties then I'm not sure a controller makes sense.  We should discuss as a group.

Reply:
So reset works now by just setting the controller's desiredDuration to Duration() as shown in the tests. This behavior cannot be built (to my knowledge, with the exception of a global Picker.of(context) which I don't think makes sense) without using a controller. What was your thought on how we'd implement reset if we were to not have a controller?